Reference/Events/EditorModel-Events/Customizing-tabs.md

```diff
@@ -0,0 +1,75 @@
+# Customizing tabs
+
+All of the current four events allow you to customize the tabs. This could be as simple as changing the label (text) of the tab - here for content:
+
+```C#
+EditorModelEventManager.SendingContentModel += (sender, e) => {
+
+    // Iterate through the tabs
+    foreach (var tab in e.Model.Tabs)
+    {
+
+	    // Change the label of the "Content" tab to "Hello World"
+        if (tab.Label == "Content")
+        {
+            tab.Label = "Hello World";
+        }
+
+    }
+
+};
+```
+
+In this Umbraco, we iterate over the tabs provided in the model, and change the label of the **Content** tab to **Hello World**.
```

Comment:
```suggestion
In Umbraco, we iterate over the tabs provided in the model, and change the label of the **Content** tab to **Hello World**.
```

Reply:
I guess my mind was full of Umbraco. It should have been "In this example". Should be fixed now ;)